README.md

```diff
@@ -180,13 +179,17 @@ results = await asyncio.gather(
 )
 
 # Compute rewards
-per_category_rewards = defaultdict(list)
-for row, result in zip(test_ds, results, strict=True):
-    reward_info = RewardFunctionInfo.model_validate(row["solution"])
-    yhat = extract_answer_loose(result[0].text)
-    reward = EVAL_FUNCTIONS[reward_info.fxn_name](yhat=yhat, y=reward_info.answer_info)
-    per_category_rewards[get_problem_category(reward_info.problem_type)].append(reward)
+rewards = accuracy_reward(
```

Comment:
Does this simplify logic or make the logic more complex?

Reply:
Just based on the fact we're replacing 6 lines with 10

Reply:
Nevermind, this is simpler

Reply:
Oh it's an interesting comment, I see what you mean. I vibe coded an `ast` analyzer and found the statements/code line actually goes down from 0.75 to 0.62, so the new code was more lines but less dense.

I decided you're right in that it's probably easier to understand with "lower level" code (not the `accuracy_reward` that handles details behind the scenes), so I went with `test=True` in the original code